base/applications/rapps/loaddlg.cpp

```diff
@@ -590,7 +609,10 @@ DWORD WINAPI CDownloadManager::ThreadFunc(LPVOID param)
         Item = GetDlgItem(hDlg, IDC_DOWNLOAD_PROGRESS);
         if (Item)
         {
+            SendMessageW(Item, WM_SETTEXT, 0, (LPARAM) L"");
             SendMessageW(Item, PBM_SETPOS, 0, 0);
+            SetWindowLongPtr(Item, GWL_STYLE, GetWindowLongPtr(Item, GWL_STYLE) & ~PBS_MARQUEE);
```

Comment:
GetWindowLongPtr might fail and return `NULL`.

Reply:
Ok, I'll make a function for this thing.